<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContinuousCaptureQt</class>
 <widget class="QMainWindow" name="ContinuousCaptureQt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>750</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>430</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ContinuousCaptureQt - Example Application</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/../Common/mv.ico</normaloff>
    <normalon>:/Common/mv.ico</normalon>:/../Common/mv.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>391</width>
       <height>22</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentText">
      <string extracomment="Select a device"/>
     </property>
    </widget>
    <widget class="QPushButton" name="open_button">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>151</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Open Selected Device</string>
     </property>
    </widget>
    <widget class="QPushButton" name="close_button">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>70</y>
       <width>161</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Close Selected Device</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="text_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>391</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Please Select A Device:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="icon_label">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>70</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="ContinuousCaptureQt.qrc">:/bulb-off.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="acquisition_button">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>70</y>
       <width>161</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Start Acquisition</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_StatusMessage">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>10</y>
       <width>361</width>
       <height>51</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QSlider" name="sliderExposureTime">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>80</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_exposureSlider">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>110</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Exposure Time [us]</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_image">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>800</width>
      <height>591</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>600</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ContinuousCaptureQt.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>comboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>ContinuousCaptureQt</receiver>
   <slot>userSelectedDeviceInComboBox()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>open_button</sender>
   <signal>clicked()</signal>
   <receiver>ContinuousCaptureQt</receiver>
   <slot>openSelectedDevice()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>close_button</sender>
   <signal>clicked()</signal>
   <receiver>ContinuousCaptureQt</receiver>
   <slot>closeSelectedDevice()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>acquisition_button</sender>
   <signal>clicked()</signal>
   <receiver>ContinuousCaptureQt</receiver>
   <slot>startAcquisition()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>500</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>374</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openSelectedDevice()</slot>
  <slot>closeSelectedDevice()</slot>
  <slot>userSelectedDeviceInComboBox()</slot>
  <slot>startAcquisition()</slot>
 </slots>
</ui>
